shared/src/packages/users/libs/types/user-delete-response-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type UserDeleteResponseDto = {
```

Comment:
check responses in other places with delete functionality in our project, as I remember we return `isDeleted: boolean` instead of id